src/System.Management.Automation/engine/Utils.cs

```diff
@@ -2099,6 +2099,13 @@ public static class InternalTestHooks
 
         internal static bool ThrowExdevErrorOnMoveDirectory;
 
+        // To emulate OneDrive behavior we use the hard-coded symlink.
+        // If OneDriveTestRecuseOn is false then the symlink works as regular symlink.
+        // If OneDriveTestRecuseOn is true then we resurce into the symlink as OneDrive should work.
```

Comment:
Should `OneDriveTestRecuseOn` be `OneDriveTestRecurseOn`?

Reply:
Opss... Fixed.